README.md

```diff
@@ -109,7 +117,7 @@ Write this grammar using EBNF with common extensions
 
 ## A2.3
 
-«replace this with your answer»
+FSM.png
```

Comment:
What you've produced is a railroad diagram for the syntax. A state machine consists of a set of states, joined by labelled transitions. When in a particular state, if an event occurs, the machine finds the transition with that label and goes to the state it points to:

![image](https://user-images.githubusercontent.com/10648/78618461-43dcb800-7840-11ea-9289-055499b1afec.png)


# -6